### DISCUSSION TITLE
Class "Spatie\WebhookClient\Jobs\ProcessWebhookJob" not found

### DISCUSSION THREAD

**Discussion by anandmainali** [spatie/laravel-webhook-client#162] [Q&A]
Everything is working on my local but when i tried to deploy in server Class "Spatie\WebhookClient\Jobs\ProcessWebhookJob" not found error occurs.

Here's the detail logs:

[stacktrace]
#0 /var/www/project/vendor/composer/ClassLoader.php(571): include()
#1 /var/www/project/vendor/composer/ClassLoader.php(428): Composer\\Autoload\\includeFile()
#2 [internal function]: Composer\\Autoload\\ClassLoader->loadClass()
#3 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(97): unserialize()
#4 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(253): Illuminate\\Queue\\CallQueuedHandler->getCommand()
#5 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php(213): Illuminate\\Queue\\CallQueuedHandler->failed()
#6 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php(192): Illuminate\\Queue\\Jobs\\Job->failed()
#7 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(581): Illuminate\\Queue\\Jobs\\Job->fail()
#8 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(527): Illuminate\\Queue\\Worker->failJob()
#9 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(455): Illuminate\\Queue\\Worker->markJobAsFailedIfWillExceedMaxAttempts()
#10 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(432): Illuminate\\Queue\\Worker->handleJobException()
#11 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(378): Illuminate\\Queue\\Worker->process()
#12 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(172): Illuminate\\Queue\\Worker->runJob()
#13 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(117): Illuminate\\Queue\\Worker->daemon()
#14 /var/www/project/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(101): Illuminate\\Queue\\Console\\WorkCommand->runWorker()
#15 /var/www/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\\Queue\\Console\\WorkCommand->handle()
#16 /var/www/project/vendor/laravel/framework/src/Illuminate/Container/Util.php(40): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#17 /var/www/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(93): Illuminate\\Container\\Util::unwrapIfClosure()
#18 /var/www/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(37): Illuminate\\Container\\BoundMethod::callBoundMethod()
#19 /var/www/project/vendor/laravel/framework/src/Illuminate/Container/Container.php(653): Illuminate\\Container\\BoundMethod::call()
#20 /var/www/project/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Illuminate\\Container\\Container->call()
#21 /var/www/project/vendor/symfony/console/Command/Command.php(298): Illuminate\\Console\\Command->execute()
#22 /var/www/project/vendor/laravel/framework/src/Illuminate/Console/Command.php(121): Symfony\\Component\\Console\\Command\\Command->run()
#23 /var/www/project/vendor/symfony/console/Application.php(1028): Illuminate\\Console\\Command->run()
#24 /var/www/project/vendor/symfony/console/Application.php(299): Symfony\\Component\\Console\\Application->doRunCommand()
#25 /var/www/project/vendor/symfony/console/Application.php(171): Symfony\\Component\\Console\\Application->doRun()
#26 /var/www/project/vendor/laravel/framework/src/Illuminate/Console/Application.php(94): Symfony\\Component\\Console\\Application->run()
#27 /var/www/project/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(129): Illuminate\\Console\\Application->run()
#28 /var/www/project/artisan(37): Illuminate\\Foundation\\Console\\Kernel->handle()
#29 {main}
"}

**Comment by EnelKyss** [comment]
I have the same issue. In laravel 9 this job is undeclared.
![image](https://user-images.githubusercontent.com/87972902/205745526-dd23d0b4-a8cf-402c-9ef2-9fe70f3e95bc.png)
![image](https://user-images.githubusercontent.com/87972902/205745632-c5ccb89c-58d9-471b-b65c-f53fe0d027dd.png)

**Comment by EnelKyss** [comment]
Oh yeah, here is why...
![image](https://user-images.githubusercontent.com/87972902/205748582-cb15d064-79f9-4fe8-9a18-167e59612d34.png)
Please fix it

**Reply by anandmainali** [reply]
@EnelKyss can you point out what is the issue? and also create PR to fix the issue?

**Comment by freekmurze** [comment]
The tests indicate the package is working correctly. Feel free to send a PR should you find that the package is in error.

**Comment by Mullersen** [comment]
I had the same issue and found a workaround here: https://github.com/spatie/laravel-webhook-client/discussions/133#discussioncomment-2212947
